spec.bs

```diff
@@ -1431,11 +1435,16 @@ an [=interest group=] |ig|, a [=boolean=] |isComponentAuction| and a [=boolean=]
   The <dfn method for="InterestGroupReportingScriptRunnerGlobalScope">sendReportTo(|url|)</dfn>
   method steps are:
 
-  1. [=exception/Throw=] a {{TypeError}} and clears report url if any of the following conditions hold:
-    * The API is called more than once;
-    * |url|'s [=url/scheme=] is not "`https`".
-  1. If called in reportWin(), set reportWinURL to url.
-  1. If called in reportResult(), set reportResultURL to url.
+  1. If [=this=]'s [=relevant global object=]'s
+    [=InterestGroupReportingScriptRunnerGlobalScope/report url=] is not null:
+    1. Set [=this=]'s [=relevant global object=]'s
+      [=InterestGroupReportingScriptRunnerGlobalScope/report url=] to null.
+    1. [=exception/Throw=] a {{TypeError}}.
+  1. Let |parsedUrl| be the result of running the [=URL parser=] on |url|.
+  1. If |parsedUrl| is failure, or |parsedUrl|'s [=url/scheme=] is not "`https`":
+    1. [=exception/Throw=] a {{TypeError}}.

```

Comment:
You can collapse this to live on the previous line if you want. It might be easier to read and is certainly common spec-wise.